app/scripts/services/brackets.coffee

```diff
@@ -17,16 +17,12 @@ angular.module('edudashAppSrv').service 'bracketsSrv', ->
     else
       switch metric
 
-        when 'AVG_MARK' then switch
-          when 0 <= val < 40 then 'POOR'
-          when 40 <= val <= 60 then 'MEDIUM'
-          when 60 < val <= 100 then 'GOOD'
-          else 'UNKNOWN'
+        when 'AVG_MARK' then throw new Error "AVG_MARK shall not be bracket"
 
         when 'AVG_GPA' then switch
-          when 1 <= val <= 3 then 'POOR'
+          when val <= 3 then 'GOOD'
           when 3 < val <= 4.2 then 'MEDIUM'
-          when 4.2 < val <= 5 then 'GOOD'  # what's the upper limit?
```

Comment:
I really think we should validate this range for min/max. Right now either the spec is wrong or we have invalid values.
If the spec is wrong, we need the right spec
If the values are invalid, they should be `UNKNOWN`, or throw an error, but not be put in a valid bracket.